rtt/base/BufferLocked.hpp

```diff
@@ -130,10 +139,14 @@ namespace RTT
                 ++itl;
             }
             // this is in any case the number of elements taken from items.
+            size_t writtenSamples = itl - items.begin(); 
```

Comment:
> should be fixed now, how do I get Travis to build ? 

Sorry, we disabled the Travis CI builds a while ago, because due to the resource limits in the free plan (either in terms of memory or in terms of build time depending on the number of parallel jobs) build jobs were killed and caused too many failed builds even if there was no actual problem. I reenabled the GitHub service hook now and also switched to the relatively new [Trusty build enviroment](https://docs.travis-ci.com/user/trusty-ci-environment/) for the master and toolchain-2.9 branches (see commit https://github.com/orocos-toolchain/rtt/commit/9bd82eb25884817e9dc5c9f572a9b716ebb205f0). Travis seems to be much faster now since [an upgrade](https://blog.travis-ci.com/2015-11-27-moving-to-a-more-elastic-future) that happened in late 2015.

Your last commit to master seems to fix the sign-compare warning:
https://travis-ci.org/orocos-toolchain/rtt/builds/102463549